src/nft/components/collection/CollectionNfts.css.ts

```diff
@@ -5,19 +5,30 @@ import { sprinkles } from '../../css/sprinkles.css'
 export const assetList = style([
   sprinkles({
     display: 'grid',
-    gap: { sm: '8', md: '12', lg: '20' },
+    gap: { sm: '8', md: '8', lg: '12', xl: '16', xxl: '20', xxxl: '20' },
   }),
   {
-    paddingLeft: 14,
-    paddingRight: 14,
-    gridTemplateColumns: 'repeat(auto-fill, minmax(160px, 1fr) )',
+    //This treatment of the grid still uses minmax, but enforces an amount of grid items per breakpoint. This means that when the bag and filter panels appear, we no longer get layout thrash and have a consistent animation as the width changes. It uses calc() and subtracts the grid gap to ensure the min size will always fit without wrapping.
+    gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/2), 1fr) )',
     '@media': {
-      'screen and (min-width: 708px)': {
-        gridTemplateColumns: 'repeat(auto-fill, minmax(240px, 1fr) )',
+      'screen and (min-width: 768px)': {
+        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/3 - 8px), 1fr) )',
       },
-      'screen and (min-width: 1185px)': {
-        gridTemplateColumns: 'repeat(auto-fill, minmax(1fr, 280px) )',
+      'screen and (min-width: 1024px)': {
+        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/3 - 12px), 1fr) )',
+      },
+      'screen and (min-width: 1280px)': {
+        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/4 - 16px), 1fr) )',
+      },
+      'screen and (min-width: 1536px)': {
+        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/5 - 20px), 1fr) )',
+      },
+      'screen and (min-width: 1920px)': {
+        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/7 - 20px), 1fr) )',
```

Comment:
I like this, but would pref themed pixel values. `src/theme/index` has `BREAKPOINTS`--could export that to make it available to sprinkles, or refactor this to be a `styled-component` and read it off the theme.

Reply:
Added common breakpoints inline.